tox.ini

```diff
@@ -25,11 +25,10 @@ deps =
 setenv =
   DJANGO_SETTINGS_MODULE=config.settings.test
   PYTHONPATH={toxinidir}/cloudigrade
-passenv =
-  AWS_SECRET_ACCESS_ID
-  AWS_SECRET_ACCESS_KEY
-  AWS_SQS_ACCESS_KEY_ID
-  AWS_SQS_SECRET_ACCESS_KEY
+  AWS_SECRET_ACCESS_ID=foo
```

Comment:
Can we just put these in `config.settings.test` so tox doesn't accidentally slurp in credentials from the env? (I assume it'll use either bogus creds if one isn't present in the env)

Reply:
See my previous comment. 

> we can't just set defaults in `test.py` because they need to be present when `base.py` loads which happens before `test.py`.

Reply:
Sounds like the problem we had before with rabbitmq url, in which case we just copied those bits to `test.py`. While that does "duplicate" code, it's a known problem, vs potentially adding a new one to the tox file.

Or am I misunderstanding something?

Reply:
This is different because now in base we need credentials to be present when we get the frozen credentials from the boto session so we can ensure we can build the SQS URL. 

I could move that functionality out into a new “dev/local”.py settings file and then it could be duplicated and changed in test and prod settings as needed, but that seems a little bit overkill to avoid a few tox environment variables since the variables have to exist in every other non-test environment anyway.